drivers/ethernet/Kconfig.xlnx_gem

```diff
@@ -0,0 +1,1387 @@
+	  64 bytes.
+
+config ETH_XLNX_GEM_PORT_0_TXBD_COUNT
+	int "Number of DMA TX data buffers and descriptors"
+	depends on ETH_XLNX_GEM_PORT_0
+	default 32
+	range 16 256
+	help
+	  Sets the number of TX packet data buffers and corresponding buffer
+	  descriptors in memory which will be accessed by the GEM via DMA.
+
+config ETH_XLNX_GEM_PORT_0_TX_BUFFER_SIZE
+	int "Transmit buffer size (in bytes)"
+	depends on ETH_XLNX_GEM_PORT_0
+	default 1536
+	range 64 16320
+	help
+	  Sets the size of the TX packet data buffers in memory which will 
+	  be accessed by the GEM via DMA. The value specified here will be
+	  adjusted to match the buffer alignment requirements of the
+	  controller.
+
+choice
+	prompt "Hardware RX buffer memory size"
+	depends on ETH_XLNX_GEM_PORT_0
+	help
+	  Configures which address space within the hardware RX buffer shall
+	  be used, effectively determining its size.
+
+config ETH_XLNX_GEM_PORT_0_HWRX_BUFFER_SIZE_FULL
+	bool "Use the full RX buffer size (8 kB)"
+
+config ETH_XLNX_GEM_PORT_0_HWRX_BUFFER_SIZE_4KB
+	bool "Disable highest address bit (4 kB)"
+
+config ETH_XLNX_GEM_PORT_0_HWRX_BUFFER_SIZE_2KB
+	bool "Disable top two address bits (2 kB)"
+
+config ETH_XLNX_GEM_PORT_0_HWRX_BUFFER_SIZE_1KB
+	bool "Disable top three address bits (1 kB)"
+
+endchoice
+
+config ETH_XLNX_GEM_PORT_0_HWRX_BUFFER_OFFSET
+	int "Data offset in the hardware RX packet buffer (in bytes)"
+	depends on ETH_XLNX_GEM_PORT_0
+	range 0 3
+	default 0
+
+choice
+	prompt "Hardware TX packet buffer memory size"
+	depends on ETH_XLNX_GEM_PORT_0
+	help
+	  Configures if the full size of half size hardware TX buffer shall
+	  be used.
+
+config ETH_XLNX_GEM_PORT_0_HWTX_BUFFER_SIZE_FULL
+	bool "Use the full TX buffer size (4 kB)"
+
+config ETH_XLNX_GEM_PORT_0_HWTX_BUFFER_SIZE_HALF
+	bool "Use half of the available TX buffer size (2 kB)"
+
+endchoice
+
+choice
+	prompt "AMBA AHB Data bus width"
+	depends on ETH_XLNX_GEM_PORT_0
+	help
+	  Sets the AMBA AHB bus width.
+
+config ETH_XLNX_GEM_PORT_0_AMBAAHB_32BIT
+	bool "32 bits"
+	help
+	  AMBA AHB bus width is 32 bits.
+
+config ETH_XLNX_GEM_PORT_0_AMBAAHB_64BIT
+	bool "64 bits"
+	help
+	  AMBA AHB bus width is 64 bits.
+
+config ETH_XLNX_GEM_PORT_0_AMBAAHB_128BIT
+	bool "128 bits"
+	help
+	  AMBA AHB bus width is 128 bits.
+
+endchoice
+
+choice
+	prompt "AHB fixed burst length for DMA data operations"
+	depends on ETH_XLNX_GEM_PORT_0
+	help
+	  Sets the AHB fixed burst length for DMA data operations.
+
+config ETH_XLNX_GEM_PORT_0_AHBBURST_SINGLE
+	bool "Always use SINGLE AHB bursts"
+
+config ETH_XLNX_GEM_PORT_0_AHBBURST_INCR4
+	bool "Attempt to use INCR4 AHB bursts"
+
+config ETH_XLNX_GEM_PORT_0_AHBBURST_INCR8
+	bool "Attempt to use INCR8 AHB bursts"
+
+config ETH_XLNX_GEM_PORT_0_AHBBURST_INCR16
+	bool "Attempt to use INCR16 AHB bursts"
+
+endchoice
+
+config ETH_XLNX_GEM_PORT_0_DISC_RX_AHB_UNAVAIL
+	bool "Discard RX packets when no AHB resource is available"
+	depends on ETH_XLNX_GEM_PORT_0
+	default n
```

Comment:
All the options are disabled by default, so there is no need to mention this line. Could you remove this one and all the other `default n` lines below.

Reply:
Just removed all the "default n" lines while fixing copy&paste mistakes both in the driver sources and the KConfig relating to two configuration items per controller.